core/src/Cuda/Kokkos_Cuda_Instance.hpp

```diff
@@ -35,8 +35,8 @@ struct CudaTraits {
     KernelArgumentLimit = 0x001000 /*  4k bytes */
   };
 
-  typedef unsigned long
-      ConstantGlobalBufferType[ConstantMemoryUsage / sizeof(unsigned long)];
+  using ConstantGlobalBufferType =
+      long[ConstantMemoryUsage / sizeof(unsigned long)];
```

Comment:
does it matter that this changed from an unsigned long to a long?

Reply:
That should not have changed. Thanks for pointing out!

Reply:
Fixed.